### ISSUE TITLE
Run pre-installed Windows from PC's SSD

### ISSUE THREAD

**Issue by GoEdgar** [utmapp/UTM#6845]
**Describe the issue**  
I have already installed Windows 10 on my SSD, which is being used on my PC. Is there any way to run UTM from this external drive or somehow to create VM from pre-installed Windows?

**Comment by Luis10110** [comment]
No, you need download windows preinstalled for vm from Microsoft web
